starter_kits/C/src/MyBot.c

```diff
@@ -0,0 +1,324 @@
+// First off, here are some defines and includes. I forget what exactly needs what
+#include <stdio.h>
+#include <stdlib.h>
+
+// We include time.h so we can seed the RNG for default "smart" movement with time()
+#include <time.h>
+
+
+
+// Now for our data structures. These are intentionally left rather basic, so add whatever you want to them
+struct ship{
```

Comment:
Please add a space before `{`

Reply:
I found a C++ linter from Google, I imagine it'll work for C. I'll try my best to lint this.

Reply:
Ended up using this linter: https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/scripts/checkpatch.pl